presto-spi/src/main/java/com/facebook/presto/spi/function/SqlFunctionProperties.java

```diff
@@ -59,6 +63,11 @@ public boolean isLegacyTimestamp()
         return legacyTimestamp;
     }
 
+    public boolean isLegacyMapSubscript()
+    {
+        return legacyMapSubscript;
+    }
+
     @Override
     public boolean equals(Object o)
```

Comment:
Could you update `equals` and `hashCode` methods as well?

Reply:
@highker 

> Could you update equals and hashCode methods as well?

Thanks for the reminder. Done.